jabkit/src/test/resources/org/jabref/cli/Chocolate.bib

```diff
@@ -0,0 +1,226 @@
+@Article{Corti_2009,
```

Comment:
This should be available in a different way --> We should avoid having duplications.

You can load it as stream if reuied.

    BibDatabase.class.getResourceAsStream("/Chocolate.bib")

In org.jabref.logic.pseudonymization.PseudonymizationTest#pseudonymizeLibrary a different file is used, because the **output** of the test is checked. In case `Chocolate.bib` updates, the output also would need an update which is unnecessary effort.


---

But I think, this can be removed, because we don't need that strong tests here.